package.json

```diff
@@ -35,7 +35,7 @@
   },
   "dependencies": {
     "@aragon/connect-agreement": "^0.0.0-beta.4",
```

Comment:
Should we be upgrading agreement as well?

Reply:
I didnt know if it would break something, that's why i didnt do it on this PR

Reply:
But yeah! Should I do it in this one?

Reply:
I'm just going to check a few things, I'm seeing errors on mainnet with this change.